galaxy_ng/app/api/ui/urls.py

```diff
@@ -84,6 +82,10 @@
         "remotes/",
         viewsets.ContainerRemoteViewSet.as_view({'get': 'list', 'post': 'create'}),
         name='execution-environments-remote-list'),
+    path(
+        "repositories/<str:base_path>/sync/",
+        views.ContainerSyncRemoteView.as_view(),
+        name='create'),
```

Comment:
The `name` here is helpful to be very descriptive so it can be used in the future to lookup the path of this url, maybe `container-repository-sync`